apps/spruce/src/pages/image/PackagesTable/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { useMemo, useRef, useState } from "react";
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import Pagination from "@leafygreen-ui/pagination";
+import {
+  useLeafyGreenTable,
+  LGColumnDef,
+  ColumnFiltersState,
+} from "@leafygreen-ui/table";
+import { BaseTable } from "components/Table/BaseTable";
+import { size } from "constants/tokens";
+import { useToastContext } from "context/toast";
+import {
+  Package,
+  ImagePackagesQuery,
+  ImagePackagesQueryVariables,
+} from "gql/generated/types";
+import { IMAGE_PACKAGES } from "gql/queries";
+
+type PackagesTableProps = {
+  imageId: string;
+};
+
+const PAGE_SIZE = 10;
```

Comment:
There's an existing constant called `DEFAULT_PAGE_SIZE` which you can import here